portal-api/src/scheduler/jobs/helpers/utilisation-report-helpers.js

```diff
@@ -44,13 +44,22 @@ const getReportOverdueChaserDate = async () => {
 };
 
 /**
- * Returns the month (1-indexed) and year of the current report period (i.e. the previous month)
- * @returns {{ month: number, year: number }}
+ * Returns the start and end months (1-indexed) and years of the current report period (i.e. the previous month)
+ * @returns {{ start: {month: number, year: number}, end: {month: number, year: number} }}
```

Comment:
Could use the following now to keep this consistent with dtfs-central-api
```
* @returns {{ start: MonthAndYear, end: MonthAndYear }}
```

Reply:
Or even
```
* @returns {ReportPeriod}
```
if we define that type too.

Reply:
`portal-api` didn't have much in the way of types, so I've added a new directory for them :)